GAME/TOMB4FX.C

```diff
@@ -126,4 +130,68 @@ void SetScreenFadeIn(long fadespeed)//34B48(<), 35048(<) (F)
 	}
 
 	return;
+}
+
+int GetFreeDrip()
+{
+	S_Warn("[GetFreeDrip] - Unimplemented!\\n");
+
+#if 0
```

Comment:
Placeholder asm code like this is no longer permitted to be merged into the master branch. Currently I'm the biggest offender and have been working to clean up the mess I've caused.